src/util.rs

```diff
@@ -152,6 +154,50 @@ pub fn format_secret(session: &Session,
     }
 }
 
+pub(crate) fn format_secret_zbus(
+    session: &Session,
+    secret: &[u8],
+    content_type: &str
+    ) -> ::Result<SecretStructInput>
+{
+    let session_path = zvariant::ObjectPath::try_from(session.object_path.to_string()).expect("remove this");
```

Comment:
This seems very unnecessary allocation/transformation. `OwnedObjectPath` implements `Deref<ObjectPath>`.

Reply:
Ah, this is because `session.object_path` is from the `dbus` library. I've only ported the `Item` stuff so far.

Reply:
Ah ok. In that case just directly create an `OwnedObjectPath` here but I'd suggest adding a `FIXME` comment here for the time when you move `session` to zbus as well.

Reply:
Unless you've a good memory (unlike me). :)